conda_project/cli/main.py

```diff
@@ -40,12 +41,111 @@ def cli() -> ArgumentParser:
 
     subparsers = p.add_subparsers(metavar="command", required=True)
 
+    _create_create_parser(subparsers, common)
+    _create_lock_parser(subparsers, common)
     _create_prepare_parser(subparsers, common)
     _create_clean_parser(subparsers, common)
 
     return p
 
 
+def _create_create_parser(
+    subparsers: "_SubParsersAction", parent_parser: ArgumentParser
+) -> None:
+    """Add a subparser for the "create" subcommand.
+
+    Args:
+        subparsers: The existing subparsers corresponding to the "command" meta-variable.
+        parent_parser: The parent parser, which is used to pass common arguments into the subcommands.
+
+    """
+    desc = "Create a new project"
+
+    p = subparsers.add_parser(
+        "create", description=desc, help=desc, parents=[parent_parser]
+    )
+    p.add_argument(
+        "-n", "--name", help="Name for the project.", action="store", default=None
+    )
+    p.add_argument(
+        "-c",
+        "--channel",
+        help=(
+            "Additional channel to search for packages. The default channel is 'defaults'. "
+            "Multiple channels are added with repeated use of this argument."
```

Comment:
Just wondering, does `conda-project` allow you to set the channel ordering list explicitly, including an explicit `defaults` as `anaconda-project` now does? If so, great!

Reply:
I think I understand your question, and yes. For example these two commands will produce different locks

```
conda project create -c defaults -c conda-forge python=3.9 sqlalchemy
```

```
conda project create -c conda-forge -c defaults python=3.9 sqlalchemy
```

Reply:
If you do `conda project create -c conda-forge python=3.9 sqlalchemy` the defaults channel will NOT be included.

That means this command will fail since `conda-token` is not in conda-forge

```
conda project create -c conda-forge python=3.9 conda-token
```

Reply:
Thanks for clarifying. I was thinking more of how `anaconda-project` can now get rid of the default of `defaults` by specifying `nodefaults`. However, it looks like `conda-project` is not taking this approach (which is fine, as it is explicit in a different way).

In addition to the locks being different, I assume your two `conda project create` commands above would list the channels in the specified order under the `channels` key in the `conda-project.yml`?